provider/bluecat/bluecat.go

```diff
@@ -1008,3 +1029,16 @@ func expandZone(zone string) string {
 	}
 	return ze
 }
+
+//extractOwnerFromTXTRecord takes a single text property string and returns the owner after parsing theowner string.
+func extractOwnerfromTXTRecord(propString string) (string, error) {
+	if len(propString) == 0 {
+		return "", errors.Errorf("External-DNS Owner not found")
+	}
+	re := regexp.MustCompile(`external-dns/owner=[a-zA-Z]+`)
```

Comment:
Looking at the  examples in our tests, it looks like the property string is separated by `,`'s so I think this would be a more appropriate regex.
```suggestion
	re := regexp.MustCompile(`external-dns/owner=[^,]+`)
```

I'm also not positive how this works with the suffix and prefix options for the txt record, but I think we should know that and document it in a separate PR.
Ref: https://github.com/kubernetes-sigs/external-dns/blob/master/pkg/apis/externaldns/types.go#L477-L479

Reply:
Yeah ... I think this is an improved regular expression. Let's do with this regular expression.

Reply:
Sounds good, I committed the change @vinny-sabatini. Not sure what the suffix and prefix parts are. It is very specific to only picking up only the [external-dns/owner="something"] always.